service.go

```diff
@@ -2,6 +2,19 @@ package libnetwork
 
 import "net"
 
+type serviceTarget struct {
+	name   string
```

Comment:
Do we need name here? This is already present in `servicePorts` unless we have different names for different targets which is not true